tests/mute-common.csv

```diff
@@ -47,6 +47,8 @@ org.jetbrains.kotlin.shortenRefs.ShortenRefsTestGenerated.testExtensionForObject
 org.jetbrains.kotlin.util.KotlinVersionsTest.testVersionsAreConsistent, KT-35567,,
 org.jetbrains.kotlin.jvm.compiler.CompileKotlinAgainstJavaTestGenerated.WithAPT.testClassWithTypeParameter, KT-36448,,
```

Comment:
The YouTrack issue mentioned here is marked as fixed. I added a comment on the issue.